### ISSUE TITLE
Groups: long comments are not truncated in group activity and group activity widget

### ISSUE THREAD

**Issue by oseg** [Elgg/Elgg#10984]
Title: long comments are not truncated in group activity and group activity widget
context: elgg v2.3.3 + groups & bookmarks plugin activated

Scenario:
- Log in
- Create a group (id 49 for instance) with group activity and group bookmarks enabled
- Create a group bookmark
- Add a long comment (more than 250 chars) in this group bookmark
- Have a look in the site activity page (url: localhost/activity)
=> The long comment is truncated after 250 characters and "..." is added at the end of the truncated comment
- Have a look in the group activity page (url: localhost/groups/activity/49)
=> The long comment is NOT truncated
- Have a look in the group profile page (url: localhost/groups/profile/49/test-group)
=>The long comment is NOT truncated in the group activity widget

Suggested fix in views/default/resources/groups/activity.php of 'groups' plugin at line 10:
```
// Add missing context
elgg_push_context('activity');
```
Suggested fix in views/default/groups/profile/activity_module.php of 'groups' plugin at line 25:
```
// Add missing context
elgg_push_context('activity');
```
Suggested fix in views/default/widgets/group_activity/content.php of 'groups' plugin at line 18:
```
// Add missing context
elgg_push_context('activity');
```

**Comment by mrclay** [comment]
Sounds good. If devs use use `elgg_in_context` it shouldn't break anything.

**Comment by hypeJunction** [comment]
Context sniffing is just bad

**Comment by mrclay** [comment]
@hypeJunction still a correct fix for 2.3.x don't you think?

**Comment by hypeJunction** [comment]
Maybe. In the long run Elgg should stop truncating and sniffing

**Comment by jdalsem** [comment]
don't forget to pop context

**Comment by jeabakker** [comment]
fixed with #11030
